plan/gen_physical_plans.go

```diff
@@ -22,8 +22,12 @@ import (
 	"github.com/pingcap/tidb/types"
 )
 
+// SupportStreamAggPushDown will be removed after supporting stream aggregation on TiKV.
+// TODO: Remove it.
+var SupportStreamAggPushDown bool
```

Comment:
Don't use a global variable to control stream agg push down. We should check if the client supports it. See https://github.com/pingcap/tidb/blob/master/store/tikv/coprocessor.go#L45